source/sass/views/youtube-regrets-reporter.scss

```diff
@@ -134,6 +134,8 @@
         background: transparent;
         color: $youtube-regrets-red;
         border: 2px solid $youtube-regrets-red;
+        // have to use !important to override the Bootstrap's
+        // .w-100 { width: 100% !important; } rule to accommodate this one-off design
         width: auto !important;
```

Comment:
This is for the sign up button on

https://foundation-s-issue-6355-xnjp28.herokuapp.com/campaigns/regrets-reporter 

<img width="425" alt="image" src="https://user-images.githubusercontent.com/2896608/112393217-f73ff600-8cb7-11eb-9a2c-0331e6a8e561.png">